src/main/java/frc/robot/statemachine/superstructure/TargetChecks.java

```diff
@@ -1,11 +1,95 @@
 package frc.robot.statemachine.superstructure;
 
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import frc.robot.Robot;
+import frc.utils.math.FieldMath;
+import org.littletonrobotics.junction.Logger;
+
 public class TargetChecks {
 
 	private final Superstructure superstructure;
+	private static final String logPath = "Statemachine/IsReadyToShoot";
```

Comment:
```suggestion
	private static final String logPath = "Statemachine/TargetChecks";
```
and add the words "IsReadyToShoot" in the specific places